lua/ulx/modules/sh/timed_owoify.lua

```diff
@@ -0,0 +1,49 @@
+CFCUlxCommands.timedtrainfuck = CFCUlxCommands.timedtrainfuck or {}
+local CATEGORY_NAME = "Fun"
+local PUNISHMENT = "timedowoify"
+local HELP = "Curse a user to be owoified at random intervals and for random amounts of time."
+
+if SERVER then
+    local function timerName( ply )
+        return "CFC_ULXCommands_TimedOwoify_" .. ply:SteamID64(), "CFC_ULXCommands_TimedOwoifyTimer2_" .. ply:SteamID64()
+    end
+
+    local function newDelay()
+        return math.Rand( 5, 25 * 60 )
+    end
+    
+    local function newDuration()
+        return math.Rand( 5, 5 * 60 )
+    end 
+
+    local function enable( caller, ply )
+        local name = timerName( ply )[1]
```

Comment:
This function returns a vararg which cannot be indexed like a table
```suggestion
        local name = timerName( ply )
```